hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/StreamType.java

```diff
@@ -38,7 +39,8 @@ public enum StreamType {
             "_Balances",
             List.of("csv", "pb"),
             Duration.ofMinutes(15L)),
-    RECORD(RecordFile::new, "recordstreams", "record", "", List.of("rcd"), Duration.ofSeconds(2L));
+    RECORD(RecordFile::new, "recordstreams", "record", "", List.of("rcd"), Duration.ofSeconds(2L)),
+    BLOCK(BlockFile::new, "blockstreams", "block", "", List.of("blk"), Duration.ofSeconds(1L));
```

Comment:
block files don't have path or nodePrefix. A 500ms close interval is closer to what's in preview, though by no means it's fixed.

```suggestion
    BLOCK(BlockFile::new, "", "", "", List.of("blk"), Duration. ofMillis(500L));
```

Reply:
Got it, fixed.